Header header

int16 agent_no
int32 id
float32 accum_distance
sensor_msgs/PointCloud2 cloud
geometry_msgs/Pose odom
geometry_msgs/Pose initial
